hpilo.py

```diff
@@ -427,7 +429,13 @@ def close(self):
             raise IloCommunicationError("Unable to resolve %s" % self.hostname)
 
         try:
-            return ssl.wrap_socket(sock, ssl_version=self.ssl_version)
+            cert_reqs=ssl.CERT_NONE
```

Comment:
Please add CERT_NONE/CERT_REQUIRED to the backwards compatibility shim at the top too for compatibility with older python versions.